src/api/routes/articleRouter.js

```diff
@@ -79,17 +86,17 @@ articleRouter.post('/:slug/highlight',
   checkValidToken,
   checkArticle.getArticle,
   validateInputs('highlight', highlightFields),
-  articleController.highlight);
+  highlightController.highlight);
 
-articleRouter.get('/:slug/highlight', checkArticle.getArticle, articleController.getHighlight);
+articleRouter.get('/:slug/highlight', checkArticle.getArticle, highlightController.getHighlight);
 
 articleRouter.post('/:slug/report',
   checkValidToken,
   bodyVerifier,
   checkArticle.getArticle,
   validateInputs('reportArticle', ['reason']),
-  checkArticleOwner.checkOwner,
```

Comment:
@truestbyheart why remove this: `checkArticleOwner.checkOwner`?

Reply:
This middleware was to check if the user reporting the article was the owner of the article... and this was defeating the purpose of allowing normal users from reporting the article.
That's why we removed it.